content/learn-more/_index.html

```diff
@@ -0,0 +1,169 @@
+          <p> Get in contact with RAPIDS, enhanced support with NVIDIA enterprise services, and the broader developer community. <a href="#get-involved">Go To Section <i class="fa-solid fa-arrow-up-right"></i></a></p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="bottom-slant-container">
+    <div class="bottom-slant-down has-background-white"></div>
+  </div>
+
+  <!-- About  -->
+  <div id="about"></div>
+  <section class="section padding-top-slant padding-bottom-slant">
+    <div class="container">
+      <h1 class="mb-5"> About The Project</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-house-laptop"></i> Familiar </h2>
+          <p>Utilizing NVIDIA CUDA primitives for low-level compute optimization, RAPIDS exposes GPU parallelism and high-bandwidth memory speed through user-friendly interfaces:</p>
+          <div class="content">
+            <ul> 
+              <li>Dataframe processing with Pandas like <a href="https://github.com/rapidsai/cudf" target="_blank">cuDF</a></li>
+              <li>Machine learning with Scikit-Learn like <a href="https://github.com/rapidsai/cuml" target="_blank">cuML</a></li>
+              <li>Graph processing with NetworkX like <a href="https://github.com/rapidsai/cugraph" target="_blank">cuGraph</a></li>
+              <li>Spatial analytics with GeoPandas like <a href="https://github.com/rapidsai/cuspatial" target="_blank">cuSpatial</a></li>
+              <li>Image processing with Scikit-image like <a href="https://github.com/rapidsai/cucim" target="_blank">cuCIM</a></li>
+              <li>Low level compute primitives with <a href="https://github.com/rapidsai/raft" target="_blank">RAFT</a></li>
+              <li>Signal processing with SciPy Signal like <a href="https://github.com/rapidsai/cusignal" target="_blank">cuSignal</a></li>
+              <li>Seamless cross-filtered dashboards with <a href="https://github.com/rapidsai/cuxfilter" target="_blank">cuxfilter</a></li>
+              <li>Apache Spark acceleration with <a href="https://nvidia.github.io/spark-rapids/" target="_blank">Spark RAPIDS</a></li>
+            </ul> 
+            <br>
+            and <a href="https://github.com/rapidsai" target="_blank">many more projects</a>...
+          </div>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-layer-plus"></i> Scalable </h2>
+          <p> RAPIDS delivers impressive acceleration from on a single GPU desktop system all the way to MNMG (multi-node multi-GPU) configurations on an expansive range of <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/" target="_blank">NVIDIA hardware</a>.  </p>
+          <div class="content">
+            <ul> 
+              <li>Impressive single GPU performance out of the box</li>
+              <li>MNMG with RAPIDS + Dask and RAPIDS + Spark</li>
```

Comment:
Would be good to have links for these. Slurm can go to deployment guide? Spark to SPARK rapids page. MNMG Dask to a dask page or docs.